MapboxCoreNavigation/Date.swift

```diff
@@ -1,17 +1,15 @@
 import Foundation
 
+let ISO8601Formatter: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.locale = Locale(identifier: "en_US_POSIX")
+    formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+    formatter.timeZone = TimeZone(secondsFromGMT: 0)
+    return formatter
+}()
+
 extension Date {
     var ISO8601: String {
-        return DateFormatter.ISO8601.string(from: self)
-    }
-}
-
-extension DateFormatter {
-    class var ISO8601: DateFormatter {
```

Comment:
Keeping it in the extension on DateFormatter would be slightly more idiomatic and Swifty.

```swift
static let ISO8601: DateFormatter = { ... }
```